.github/PULL_REQUEST_TEMPLATE/BugFix.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug Fix
+about: Add a bug fix this project!
+title: "[Feature] Short Form Title (50 chars or less.)"
+labels: bug
+assignees: 'WithoutPants, bnkai, Leopere'
+
+---
+<!-- Explain what your bugfix seeks to remedy in a short paragraph. -->
+# Scope
+
+<!-- Declare any issues by typing `fixes #1` or `closes #1` for example so that the automation can kick in when this is merged -->
+## This Pull Request Attempts to close the following issues
+
+<!-- Please put an X in the following items to show that you've done your due diligence. -->
+## Due Diligence Checklist
+
+* [ ] I've run Code linters and [gofmt](https://golang.org/cmd/gofmt/) to make sure that my code is readable.
+* [ ] I have read through formerly submitted pull requests and issues to make sure that this is required and so that I can manage to close any Issues needing closed relating to this feature submission.
+* [ ] To the best of my ability, I've followed the [CONTRIBUTING.md](/CONTRIBUTING.md) to make sure that I'm up to date on how to best contribute to Stash projects.
+* [ ] I  commented adequately on my code with the expectation in mind that anyone else should be able to look at this code I've submitted and know exactly what's happening and what the expectations are.
+* [ ] I acknowledge that if applicable to me, submitting and subsequent acceptance of this Pull Request I, the code contributor of this Pull Request, agree and acknowledge my understanding that the new code license has now been updated to [AGPL](/LICENSE.md). I agree that all code before this Pull Request, which I've previously submitted, is now to be re-licensed under the new license AGPL and no longer the former MIT license.
+
+  <!-- Explain why you may not have checked all of the above boxes. -->
+  ### In case I haven't checked all of the boxes above here is my explanation as to why not.
```

Comment:
I'd remove the heading here and just leave the comment, adding an "if applicable".